tests/cases/fourslash/jsDocClasses1.ts

```diff
@@ -0,0 +1,28 @@
+///<reference path="fourslash.ts" />
+
+// @allowNonTsExtensions: true

```

Comment:
do not think you needed the fourslash test here. a compiler baseline test would have been more appropriate.